Dockerfile.garage-push

```diff
@@ -0,0 +1,10 @@
+FROM advancedtelematic/aktualizr
```

Comment:
Are we planning on some process to keep this image up to date?  The last push to that image was a month ago.

Reply:
Good question. I guess libsqlite3-dev could be added and the name could be changed to "aktualizr-build-env" or something.

Reply:
The other option would be to start from one of the Ubuntu images and include all the packages required for the build

Reply:
I was under the impression that some part of the Travis CI process pushed the docker image, but that does not appear to be the case. How did this ever get updated before?